docs/running-on-kubernetes.md

```diff
@@ -537,6 +578,7 @@ from the other deployment modes. See the [configuration page](configuration.html
   <td><code>spark.kubernetes.driver.annotations</code></td>
   <td>(none)</td>
   <td>
+    <i>Deprecated.</i> Use <code>spark.kubernetes.driver.annotation.<annotationKey></code> instead.
```

Comment:
mention here that this is to support annotation values with `=` and `,` characters